boards/nucleo-l1/Makefile.include

```diff
@@ -0,0 +1,56 @@
+## the cpu to build for
+export CPU = stm32l1
+export CPU_MODEL = stm32l152ret6
+
+#define the default port depending on the host OS
+OS := $(shell uname)
+ifeq ($(OS),Linux)
+  PORT ?= /dev/ttyUSB0
```

Comment:
change this to `/dev/ttyACM0`, and the board is able to communicate out-of-the-box using the integrated USB-UART adapter without any external hw...